src/_nebari/upgrade.py

```diff
@@ -1283,13 +1497,10 @@ def _version_specific_upgrade(
         keycloak_admin = None
 
         # Prompt the user for role assignment (if yes, transforms the response into bool)
-        assign_roles = (
-            Prompt.ask(
-                "[bold]Would you like Nebari to assign the corresponding role/scopes to all of your current groups automatically?[/bold]",
-                choices=["y", "N"],
-                default="N",
-            ).lower()
-            == "y"
+        # This needs to be monkeypatched and will be addressed in a future PR. Until then, this causes test failures.
```

Comment:
I believe this has been solved in https://github.com/nebari-dev/nebari/pull/2879/files